torch/_inductor/ir.py

```diff
@@ -3219,9 +3219,19 @@ def __init__(
         )
         self.use_cpp_op_schema = False
 
-        if getattr(torch.ops.aten, kernel.__name__, None) is kernel:
+        def is_aten_op(kernel):
+            obj = torch.ops.aten
+
+            for part in kernel.__name__.split("."):
+                obj = getattr(obj, part, None)
+                if obj is None:
+                    return False
+
+            return obj is kernel
+
+        if is_aten_op(kernel):
             self.kernel = (
-                f"at::{kernel.__name__}"
+                f"at::{kernel.__name__.split('.')[0]}"
```

Comment:
Is `kernel._schema.name` the same thing you are trying to get here? Or, maybe you can use `kernel._overloadpacket` too?

Reply:
@jgong5 refactored this part so that it always operates on overloadpackets